### PR TITLE
Add 13-16 examples

### PR DESCRIPTION
**Pull request by TopIvanAbramov**

### CODE REVIEW

File: examples/14.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#Program to find the factors of a number
+
+
+routine getFactors(x: integer) : array [] boolean is
+    var factors is array [x] boolean
+
+    for i in range 1 .. x loop
```
Comment (pfh-hbq) [inline examples/14.txt]: remove range, because we don't use it as I understood

File: examples/16.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    return newList
+end
+
+
+
+var a is array [10] integer
+
+
+for i in range 1 .. a.size loop
```
Comment (pfh-hbq) [inline examples/16.txt]: remove range please :)

File: examples/16.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Program to reverse a list
+
+routine reverse(dataList: array [] integer) : array [] integer is
+    var length : integer = dataList.size
+
+    var s : integer is length
+
+    var newList is array [length] integer
```
Comment (pfh-hbq) [inline examples/16.txt]: remove is

File: examples/13.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Program to convert kilometers to miles
+
+var x : integer is 3
+
+
+routine convertToMiles(x : integer) : real is
+    var convertFactor : real = 0.621371
```
Comment (TheSharpOwl) [inline examples/13.txt]: I think it should be "is" not =

File: examples/13.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Program to convert kilometers to miles
```
Comment (TheSharpOwl) [inline examples/13.txt:1]: better to delete comments

File: examples/14.txt [hunk 6]
```diff
@@ -0,0 +1,18 @@
+#Program to find the factors of a number
```
Comment (TheSharpOwl) [inline examples/14.txt]: Here too no comments

File: examples/13.txt (hunk 5 above)
Comment (TheSharpOwl) [inline examples/13.txt]: better to delete comments

File: examples/14.txt (hunk 6 above)
Comment (matveyplevako) [inline examples/14.txt]: Remove comment

File: examples/13.txt (hunk 5 above)
Comment (matveyplevako) [inline examples/13.txt]: Remove comment

File: examples/14.txt [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+
+routine getFactors(x: integer) : array [] boolean is
```
Comment (TheSharpOwl) [inline examples/14.txt]: return a constant size. only parameter can omit the size

File: examples/14.txt [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+           factors[i] := true
+	else
+           factors[i] := false
+        end	
+    end
+
+   return factors
+end
+
+var factorsArray : array [] boolean is getFactors(10)
```
Comment (TheSharpOwl) [inline examples/14.txt]: here also there should be a size for the array

File: examples/16.txt [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# Program to reverse a list
+
+routine reverse(dataList: array [] integer) : array [] integer is
```
Comment (TheSharpOwl) [inline examples/16.txt]: here also, the return type should have a fixed size

File: examples/16.txt [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    var length : integer = dataList.size
```
Comment (TheSharpOwl) [inline examples/16.txt]: it should be "is" not "="

File: examples/16.txt [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+var a is array [10] integer
+
+
+for i in range 1 .. a.size loop
+      a[i] := i
+end
+
+
+
+var b is array [] integer is reverse(a)
```
Comment (TheSharpOwl) [inline examples/16.txt]: add a const size for the array

File: examples/16.txt [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+   return false
+end
+
+
+var math : subject
+
+math.id = 5
+math.professorId = 3
+math.credits = 0
```
Comment (matveyplevako) [inline examples/16.txt]: assign should be :=

File: examples/16.txt [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+
+var physics : subject
+
+math.id = 0
+math.professorId = 0
+math.credits = -30
```
Comment (matveyplevako) [inline examples/16.txt]: same, replace with :=

**Pull request merged**